webservices/resources/operations_log.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from flask_apispec import doc
+from webservices import args
+from webservices import docs
+from webservices import utils
+from webservices import schemas
+from webservices.common import models
+from webservices.exceptions import ApiError
+from webservices.common.views import ApiResource
+
+logger = logging.getLogger(__name__)
+
+
+@doc(tags=['filings'], description=docs.OPERATIONS_LOG)
+class OperationsLogView(ApiResource):
+    model = models.OperationsLog
+    schema = schemas.OperationsLogSchema
+    page_schema = schemas.OperationsLogPageSchema
+
+    filter_multi_fields = [
```

Comment:
Should we add indexes to the `operations_log` table to correspond with these filters?

Reply:
This is not the final endpoint, but yes i need to create one or two indexes on this endpoint. 
The endpoint keep changing based off @jwchumley feedback that i get after testing in my local env.